receiver/awscontainerinsightreceiver/internal/k8sapiserver/prometheus_scraper.go

```diff
@@ -109,6 +114,6 @@ func (ps *PrometheusScraper) Shutdown() {
 		if err != nil {
 			ps.settings.Logger.Error("Unable to shutdown SimplePrometheusReceiver", zap.Error(err))
 		}
-		ps.running = err != nil
+		ps.running = false
```

Comment:
My bad - I think I misunderstood this logic before.. if there is no err, then err != nil would be false, so running would be false. If there was an err and it didnt shutdown, err != nil would be true and it would continue to be running.

So if you wanna just keep this the way it was, thats fine as well.